tests/engine/test_arg_utils.py

```diff
@@ -365,3 +365,20 @@ def test_human_readable_model_len():
     for invalid in ["1a", "pwd", "10.24", "1.23M"]:
         with pytest.raises(ArgumentError):
             args = parser.parse_args(["--max-model-len", invalid])
+
+
+def test_ep_config():
+    parser = EngineArgs.add_cli_args(FlexibleArgumentParser())
+
+    # test JSON config
+    args = parser.parse_args([
+        "--ep-config",
+        '{"enable_eplb": true, "lb_window_size": 1500, '
+        '"lb_step_interval": 4000, "lb_log_balancedness": true,'
+        '"num_redundant_experts":10}',
+    ])
+    assert args.ep_config.enable_eplb
+    assert args.ep_config.lb_window_size == 1500
+    assert args.ep_config.lb_step_interval == 4000
+    assert args.ep_config.lb_log_balancedness
+    assert args.ep_config.num_redundant_experts == 10
```

Comment:
This test is not really necessary because it duplicates just duplicates the `json.loads` test case of https://github.com/vllm-project/vllm/blob/e6b90a2805e809022580f2c1f4928c64b5f531f1/tests/engine/test_arg_utils.py#L20-L31